pyproject.toml

```diff
@@ -8,7 +8,7 @@ description = "Protocol Buffer Validation for Python"
 readme = "README.md"
 license = { file = "LICENSE" }
 keywords = ["validate", "protobuf", "protocol buffer"]
-requires-python = ">=3.8"
```

Comment:
We should remove the corresponding line under `classifiers` too, and potentially add 3.14 (depending on the other comment).